### PR TITLE
fix: change the error prompt from Chinese to English for addon delete

### PR DESCRIPTION
**Pull request by littlejiancc**
#### What type of this PR

/kind feature

#### What this PR does / why we need it:
feature: change the error prompt from Chinese to English for addon delete

#### Which issue(s) this PR fixes:
https://erda.cloud/erda/dop/projects/387/issues/all?id=206519&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIl0sImFzc2lnbmVlSURzIjpbIjEwMDEyNjEiXX0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjF9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=506&type=BUG

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/orchestrator/services/addon/addon.go [hunk 1]
```diff
@@ -1401,7 +1401,7 @@ func (a *Addon) Delete(userID, routingInstanceID string) error {
 			return err
 		}
 		if !permissionResult.Access {
-			return errors.New("权限不足")
+			return errors.New("access denied")
```
Comment (Effet) [inline modules/orchestrator/services/addon/addon.go:1404]: it's not i18n, it just replace chinese to english?

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/orchestrator/services/addon/addon.go (hunk 1 above)
Comment (littlejiancc) [reply modules/orchestrator/services/addon/addon.go:1404]: yes, the err by errors.new just replace chinese to english
image

File: modules/orchestrator/services/addon/addon.go [hunk 2]
```diff
@@ -1411,7 +1411,7 @@ func (a *Addon) Delete(userID, routingInstanceID string) error {
 		return err
 	}
 	if count > 0 {
-		return errors.New("addon正在被引用，无法删除")
+		return errors.New("addon is being referenced,can't delete")
```
Comment (sfwn) [inline modules/orchestrator/services/addon/addon.go:1414]: ...ced, cant't

notice space between comma and alphabet.

**Review by sfwn — CHANGES_REQUESTED** [review]

**Comment by sfwn** [comment]
add unit test to match requirement of codecov.

**Review by sfwn — APPROVED** [review]

**Pull request merged**